content/en/docs/instrumentation/js/libraries.md

```diff
@@ -14,7 +51,7 @@ For example,
 will automatically create [spans](/docs/concepts/signals/traces/#spans) based on
 the inbound HTTP requests.
 
-## Setup
+### Setup
```

Comment:
I think we should remove the first part of this section, the

```
Each instrumentation library is an NPM package, and installation is typically
done like so:
``console
npm install <name-of-package>
``
```

bit. And instead just say something like:

```
Each instrumentation library can be installed via NPM. For example, here's how you can install the instrumentation-express and instrumentation-http [...]
```

Reply:
fixed